src/core/server/server.h

```diff
@@ -512,6 +518,9 @@ class Server : public ServerInterface,
   absl::BitGen bitgen_ ABSL_GUARDED_BY(mu_call_);
 
   std::list<ChannelData*> channels_;
+  absl::flat_hash_set<OrphanablePtr<ServerTransport>> connections_
```

Comment:
Related to my comment elsewhere about xDS integration, I wonder if instead of tracking this linkage here, we should instead delegate to the listeners to do that.  I believe the chttp2 listener already tracks active connections, since it needs to do that for reasons related to the xDS config -- if the xDS config changes, it may need to close existing connections.  So if we are also tracking this here, then we're going to wind up tracking it twice.

Reply:
Happy to see a design that does that going forward, but right now this is sufficient.

I'd like that design to be able to live alongside the code in server.cc and not distributed differently amongst the transports.

Reply:
Yeah, I agree, all of the common config-related code should be here, not in transport-specific code.